animation_nodes/nodes/midi/fileParser.py

```diff
@@ -0,0 +1,55 @@
+import os
```

Comment:
File names should follow the convention `file_parser.py`, and so on. This goes for all files.